src/util/orders.ts

```diff
@@ -83,3 +86,62 @@ export const buildMarketOrders = (
 
     return [ordersToFill, roundedAmounts, canBeFilled];
 };
+
+export const orderDetailsFeeEther = (
```

Comment:
Now that we moved the responsability of displaying the fees in ZRX/USD to a component (since it is something the UI does), do we need this method? I think we no longer use it and we can remove it + its corresponding tests.

Reply:
now it's removed